catalog/nsili/catalog-nsili-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -23,6 +23,9 @@
         <AD description="The URL of the IOR File to use for the NSILI CORBA Server"
             name="Ior File URL" id="iorUrl" required="true" type="String"/>
 
+        <AD description="The Location of the local IOR File to use for the NSILI CORBA Server"
```

Comment:
Don't make this a separate property - they are both URL's have to code check the scheme and then decide what to do.  We will likely also need support for FTP.